### PR TITLE
[Elastic Agent] Add hint on how to build and test the container for Elastic Agent

### PR DESCRIPTION
**Pull request by ruflin**
I had this as a local note and used it often. I thought it maybe worth sharing.

### CODE REVIEW

**Comment by ruflin** [comment]
@urso I use these kind of snippets during development. Do we have a good place to document these? I strongly prefer to have these development related snippets in the same repository as I do not want to jump through hoops.

File: x-pack/elastic-agent/docs/README.md [hunk 1]
```diff
@@ -4,3 +4,17 @@ Elastic Agent Documentation
 
 The source files for the Elastic Agent documentation are currently stored
 in the [observability-docs](https://github.com/elastic/observability-docs) repo.
+
+## Testing docker container
```
Comment (urso) [inline x-pack/elastic-agent/docs/README.md:8]: How about us moving some of the instructions/tips to the top level `elastic-agent/README.md`. Or at least have a link telling you there are some instructions?

**Comment by urso** [comment]
> Do we have a good place to document these?

Not yet. I don't mind a developer focused docs in the doc directory. But top level README.md in elastic-agent directory should be my entrypoint, either with more complete docs + TOC, and/or a list of links including top level architecture docs, build instructions, ...

File: x-pack/elastic-agent/docs/README.md (hunk 1 above)
Comment (ruflin) [reply x-pack/elastic-agent/docs/README.md:8]: As far as I know, this is the only README we have. Github shows it automatically like the top level one. Happy to move it to the top. It is also the place I was looking for it initially.

**Review by urso — APPROVED** [review]

**Pull request merged**